src/assets/images/AuthLogo/logo.svg

```diff
@@ -0,0 +1,22 @@
+<svg width="54" height="56" viewBox="0 0 54 56" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@MPucheta this file is the app logo. Do we have it already in the repo?

Reply:
Yes we do. This is the one `assets/images/logo.svg`